src/sagemaker_xgboost_container/data_utils.py

```diff
@@ -485,6 +487,52 @@ def get_recordio_protobuf_dmatrix(path, is_pipe=False):
         raise exc.UserError("Failed to load recordio-protobuf data with exception:\n{}".format(e))
 
 
+def _get_pipe_mode_files_path(data_path: Union[List[str], str]) -> List[str]:
+    """
+    :param data_path: Either directory or file
+    """
+    # For pipe mode, we leverages mlio directly by creating a list of SageMakerPipe.
```

Comment:
I'm not sure what this comment means or what its purpose is. Let's re-word it or remove it.

Reply:
will remove in next rev.